### PR TITLE
pythonPackages.pyturbojpeg: init at 1.4.1

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
https://github.com/NixOS/nixpkgs/pull/118234#discussion_r605800730

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

~I'm not done running Home Assistant's tests yet.~ Done

### CODE REVIEW

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by dotlambda** [comment]
Can someone test this on Darwin!

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 118407` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.pyturbojpeg</li>
    <li>python39Packages.pyturbojpeg</li>
  </ul>
</details>

File: pkgs/servers/home-assistant/component-packages.nix [hunk 1]
```diff
@@ -277,7 +277,7 @@
     "foscam" = ps: with ps; [ ]; # missing inputs: libpyfoscam
     "foursquare" = ps: with ps; [ aiohttp-cors ];
     "free_mobile" = ps: with ps; [ ]; # missing inputs: freesms
-    "freebox" = ps: with ps; [ aiohttp-cors netdisco zeroconf ]; # missing inputs: freebox-api
+    "freebox" = ps: with ps; [ aiohttp-cors freebox-api netdisco zeroconf ];
```
Comment (fabaff) [inline pkgs/servers/home-assistant/component-packages.nix:280]: `component-packages` should only add `pyturbojpeg`. Easier to revert if needed. The other two dependencies will be added with the Home Assistant update to 2021.4.X.

File: pkgs/servers/home-assistant/component-packages.nix [hunk 2]
```diff
@@ -448,7 +448,7 @@
     "linux_battery" = ps: with ps; [ batinfo ];
     "lirc" = ps: with ps; [ ]; # missing inputs: python-lirc
     "litejet" = ps: with ps; [ ]; # missing inputs: pylitejet
-    "litterrobot" = ps: with ps; [ ]; # missing inputs: pylitterbot
+    "litterrobot" = ps: with ps; [ pylitterbot ];
```
Comment (fabaff) [inline pkgs/servers/home-assistant/component-packages.nix:451]: Dito

**Review by fabaff — APPROVED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package built (see https://github.com/NixOS/nixpkgs/pull/118407#issuecomment-812884740)
- [x] all depending packages build
- [x] tested on x86_64

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118407` at a8c0dbda run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.pyturbojpeg</li>
    <li>python39Packages.pyturbojpeg</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Please add a comment on the line above, explaining the purpose of this patch.
Near pkgs/development/python-modules/pyturbojpeg/default.nix:22:5:
```
   |
22 |     (substituteAll {
   |     ^
```
</li>
  </ul>
</details>

**Review by mweinelt — APPROVED** [review]

**Pull request merged**